docs/maps/maps-aggregations.asciidoc

```diff
@@ -69,7 +69,7 @@ then accumulates the most relevant documents based on sort order for each entry
 
 To enable top hits:
 
-. Click *Add layer* button and select *Documents* source.
+. Click *Add layer*, then select *Documents* layer card.
 . Configure *Index pattern* and *Geospatial field*.
```

Comment:
Should this be

Configure the **Index pattern** and **Geospatial** fields?

Reply:
Its `Geospatial field`